src/posthog-core.js

```diff
@@ -280,7 +280,12 @@ PostHogLib.prototype._loaded = function () {
     // this happens after so a user can call identify in
     // the loaded callback
     if (this.get_config('capture_pageview')) {
-        this.capture_pageview()
+        // Testing, need to replace this with production code in #295 after testing
+        if (window && window.location.hostname === 'posthog.com') {
+            this.capture('$pageview', {}, {})
+        } else {
+            this.capture('$pageview')
```

Comment:
I've dug into the code, but I really don't see what the addition of `, {}, {}` changes. There's no comment explaining the mechanics of this change, and as far as I reckon, these are the default values already. Could you lay it out?

Reply:
Fixed in the PR: https://github.com/PostHog/posthog-js/pull/295#issuecomment-932347635